### PR TITLE
README/Repo: remove Travis CI + add build matrix + add Freenode qwebirc link

### PR DESCRIPTION
**Pull request by anonimal**

### CODE REVIEW

**Comment by anonimal** [comment]
As discussed with @danrmiller, if we can't use the coveralls API without a requisite of Travis CI or other non-Buildbot CI, we should remove the coveralls badge and disable the account (it will be useless).

File: README.md [hunk 1]
```diff
@@ -2,16 +2,32 @@
...
-[![Coverage Status](https://coveralls.io/repos/github/monero-project/monero/badge.svg?branch=master)](https://coveralls.io/github/monero-project/monero?branch=master)
-
 ## Development Resources
 
 - Web: [getmonero.org](https://getmonero.org)
 - Forum: [forum.getmonero.org](https://forum.getmonero.org)
 - Mail: [dev@getmonero.org](mailto:dev@getmonero.org)
 - GitHub: [https://github.com/monero-project/monero](https://github.com/monero-project/monero)
-- IRC: [#monero-dev on Freenode](irc://chat.freenode.net/#monero-dev)
+- IRC: [#monero-dev on Freenode](http://webchat.freenode.net/?randomnick=1&channels=%23monero-dev&prompt=1&uio=d4)
```
Comment (moneromooo-monero) [inline README.md]: What is uio=d4 ?

**Comment by moneromooo-monero** [comment]
ok

File: README.md (hunk 1 above)
Comment (anonimal) [inline README.md]: @moneromooo-monero I don't know. Their site generated the link.

**Comment by anonimal** [comment]
Added new commits since @moneromooo-monero's ACK.

**Comment by anonimal** [comment]
@moneromooo-monero I can close and squash if you'd like.

**Pull request merged**